tests/console/login.go

```diff
@@ -15,8 +15,8 @@ import (
 	"kubevirt.io/kubevirt/pkg/util/net/dns"
 )
 
-// LoginToFactory represents the LogIn* functions signature
-type LoginToFactory func(*v1.VirtualMachineInstance) error
+// LoginToFunction represents any of the LoginTo* functions
+type LoginToFunction func(*v1.VirtualMachineInstance) error
```

Comment:
What do you think about a plain `LoginTo`?

Reply:
I'm not sure if a reader of
```
func waitUntilVMIReady(vmi *v1.VirtualMachineInstance, loginTo console.LoginTo)
```
would immediately understand that she needs to supply a function. Also I kinda prefer to have nouns as argument, not a verb with a preposition. But I do not have a very strong opinion about it. Reviewer is King - let me know if you insist.

Reply:
I do not mind as well, it was just a thought.